crawl-ref/source/fight.cc

```diff
@@ -701,22 +701,42 @@ void get_cleave_targets(const actor &attacker, const coord_def& def,
     if (weap && item_attack_skill(*weap) == SK_AXES
             || attacker.is_player()
                && (you.form == transformation::hydra && you.heads() > 1
-                   || you.duration[DUR_CLEAVE]))
+                   || you.duration[DUR_CLEAVE])
+						|| weap && weap->sub_type == WPN_SCYTHE)
     {
```

Comment:
This line could do with being lined up with the previous `|| attacker.is_player()` line.